beetmoverscript/templates/firefox_nightly_repacks.yml

```diff
@@ -30,6 +30,11 @@ mapping:
       destinations:
         - {{ upload_date }}-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums.asc
         - latest-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums.asc
+    buildhub.json:
+      s3_key: {{ upload_date }}-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
```

Comment:
`s3_key` is solely used in checksums generation. Whatever we end up placing here, will be landing in the SHA512SUMS files (e.g. [here](https://archive.mozilla.org/pub/firefox/candidates/61.0b14-candidates/build1/SHA256SUMS)). So in this case, it's better if we simply keep the `firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json` part without its leading `upload_date ...` prefix.